allofplos/plos_corpus.py

```diff
@@ -594,9 +594,9 @@ def download_updated_xml(article_file,
     articletree_remote = et.parse(url)
     articleXML_remote = et.tostring(articletree_remote, method='xml', encoding='unicode')
     try:
-        articletree_local = et.parse(article_file)
+        articletree_local = et.parse(doi_to_path(article_file))
```

Comment:
Need to check data type. doi_to_path takes a doi as input and you are passing an "article_file" (that by the way it is a filename).